src/ws.nim

```diff
@@ -1,13 +1,13 @@
 import chronos, chronicles, httputils, strutils, base64, std/sha1, random,
-    streams, nativesockets, uri, times, chronos.timer, tables
+    streams, nativesockets, uri, times, chronos/timer, tables
 
 const
-  MaxHttpHeadersSize = 8192        # maximum size of HTTP headers in octets
-  MaxHttpRequestSize = 128 * 1024  # maximum size of HTTP body in octets
+  MaxHttpHeadersSize = 8192       # maximum size of HTTP headers in octets
+  MaxHttpRequestSize = 128 * 1024 # maximum size of HTTP body in octets
   HttpHeadersTimeout = timer.seconds(120) # timeout for receiving headers (120 sec)
-  HttpBodyTimeout = timer.seconds(12)     # timeout for receiving body (12 sec)
-  HeadersMark = @[byte(0x0D), byte(0x0A), byte(0x0D), byte(0x0A)]
   WebsocketUserAgent* = "nim-ws (https://github.com/status-im/nim-ws)"
+  CRLF* = "\c\L"
```

Comment:
```suggestion
  CRLF* = "\r\n"
```

Reply:
This still needs to be changed to the @arnetheduck suggestion.